packages/webapp/package.json

```diff
@@ -22,10 +22,12 @@
         "@testing-library/dom": "8.20.1",
         "@testing-library/jest-dom": "6.2.0",
         "@testing-library/react": "12.1.5",
+        "@types/lodash": "4.14.149",
```

Comment:
Any specific reason to use this version? I see it was released 5 years ago https://www.npmjs.com/package/@types/lodash?activeTab=versions?

As we use the latest version of lodash, shouldn't we also use the latest version of types?